package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "lint": "./node_modules/eslint/bin/eslint.js ./*.js ./assets ./deserializers ./serializers ./services ./test",
     "test": "./node_modules/mocha/bin/mocha 'test/**/*.js'",
```

Comment:
[Specs] The yarn test command seems to be broken 🤔

Reply:
What was the problem?